src/View/Product/show.html.twig

```diff
@@ -0,0 +1,53 @@
+{% extends 'layout.html.twig' %}
+
+{% block stylesheet %}<link rel="stylesheet" href="../assets/css/show.css">{% endblock %}
+
+{% block content %}
+
+<section id="product_page">
+        <article class="product_carousel">
+            <div class="product_image" id="product_imgs">
+                {% for photo in product.photo %}
+                    <div class="photo" style="background-image:url({{photo}});"></div>
+                {% endfor %}
+            </div>
+                <button id="right" class="product_btn">></button>
+                <button id="left" class="product_btn"><</button>
+        </article>
+
+        <article class="product_Info">
+            <div>
+                <h1>Nom du produit</h1>
+                <div class="product_ColorMatter"><span>Couleur:</span> <p class="pastilleColor"></p><span>Matière:</span> Bois </br><img src="planche.png" alt=""></div>
+                <div class="product_state"><span>État:</span> quasi neuf</div>
+                <div class="product_description">   blablablablabla blablabla blablablabla blablabla blablabla blabla blablablabla blablabla blablablabla blabla blaElleEstBelleLaChaisebla blabla blabla blabla blablablabla blablabla</div>
+                <div class="product_price"><span>Prix:</span> 8000€</div>
+            </div>
+        </article>
+
+        <article class="product_description2">
```

Comment:
I think a ul li display would have more sense, since it will be a list of product description